### PR TITLE
fix: Update BASE_R4_RESOURCES with missing resources

### PR DESCRIPTION
**Pull request by nguyen102**
Description of changes:
fix: Update BASE_R4_RESOURCES with missing resources

Removed 
* MedicinalProductOperation

Added 
* MedicinalProductInteraction
* SubstanceNucleicAcid

List from 
http://hl7.org/fhir/R4/valueset-resource-types.html

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Comment by rsmayda** [comment]
Do we also need to add?
- https://www.hl7.org/fhir/servicerequest.html

File: src/constants.ts [hunk 1]
```diff
@@ -374,6 +375,7 @@ export const BASE_R4_RESOURCES: R4Resource[] = [
     'StructureMap',
     'Subscription',
     'Substance',
+    'SubstanceNucleicAcid',
     'SubstancePolymer',
     'SubstanceProtein',
     'SubstanceReferenceInformation',
```
Comment (rsmayda) [inline src/constants.ts:381]: could we swap line 382 and 383 (so they can be alphabetized?

'SubstanceSourceMaterial',
'SubstanceSpecification',

Comment (nguyen102) [reply src/constants.ts:381]: done

**Comment by nguyen102** [comment]
@rsmayda I think `ServiceRequest` is already included?
https://github.com/awslabs/fhir-works-on-aws-interface/blob/fd0f127726b0726fc9e1a5f25941ff95f5106616/src/constants.ts#L116

**Comment by rsmayda** [comment]
> @rsmayda I think `ServiceRequest` is already included?
> 
> https://github.com/awslabs/fhir-works-on-aws-interface/blob/fd0f127726b0726fc9e1a5f25941ff95f5106616/src/constants.ts#L116

Looks like your right!

**Review by rsmayda — APPROVED** [review]

**Review by rsmayda — APPROVED** [review]

**Review by carvantes — APPROVED** [review]

**Pull request merged**